src/Mutate.tsx

```diff
@@ -120,10 +121,12 @@ class ContextlessMutate<TData, TError> extends React.Component<
     const { base, path, verb, requestOptions: providerRequestOptions } = this.props;
     this.setState(() => ({ error: null, loading: true }));
 
-    const requestPath =
-      verb === "DELETE" && typeof body === "string"
-        ? url.resolve(base!, url.resolve(path, body))
-        : url.resolve(base!, path);
+    let requestPath: string;
```

Comment:
I think that you can keep the ternary here, for 2 choices is fine 😃

Reply:
Changed it back to ternary :smiley: